server/config/forms/relocationAndInjuriesValidation.test.js

```diff
@@ -139,6 +141,59 @@ describe("'complete' schema", () => {
       ])
     })
 
+    it('Selecting No for Was the prisoner compliant but not selecting relocationType returns validation error message', () => {
+      const input = {
+        ...validInput,
+        relocationCompliancy: 'false',
+        relocationType: undefined,
+      }
+      const { errors, formResponse } = check(input)
+
+      expect(errors).toEqual([
+        {
+          href: '#relocationType',
+          text: 'Select the type of relocation',
+        },
+      ])
+      expect(formResponse.relocationCompliancy).toEqual(false)
+      expect(formResponse.relocationType).toBe(undefined)
+    })
```

Comment:
can we have a test for what happens when the post passes back both a type and an other

Reply:
Test added.